### PR TITLE
chore: remove version id, clean unused stats

### PR DESCRIPTION
**Pull request by Jayachand**
## What are the changes introduced in this PR?

Write a brief explainer on your code changes.

## What is the related Linear task?

Resolves DAT-1354

## Please explain the objectives of your changes below

Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here

### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### Any technical or performance related pointers to consider with the change?

N/A

@coderabbitai review

<hr>

### Developer checklist

- [ ] My code follows the style guidelines of this project

- [ ] **No breaking changes are being introduced.**

- [ ] All related docs linked with the PR?

- [ ] All changes manually tested?

- [ ] Any documentation changes needed with this change?

- [ ] Is the PR limited to 10 file changes?

- [ ] Is the PR limited to one linear task?

- [ ] Are relevant unit and component test-cases added in **new readability format**?

### Reviewer checklist

- [ ] Is the type of change in the PR title appropriate as per the changes?

- [ ] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3494/test-report.html

File: src/util/prometheus.js [hunk 1]
```diff
@@ -797,88 +550,6 @@ class Prometheus {
...
-      {
-        name: 'creation_time',
-        help: 'creation_time',
-        type: 'histogram',
-        labelNames: ['transformerVersionId', 'language', 'identifier', 'publish', 'testMode'],
-      },
-      { name: 'get_tracking_plan', help: 'get_tracking_plan', type: 'histogram', labelNames: [] },
-      { name: 'createivm_duration', help: 'createivm_duration', type: 'histogram', labelNames: [] },
-      {
-        name: 'fetchV2_call_duration',
```
Comment (kanishkkatara) [inline src/util/prometheus.js:809]: Why are we removing these? Shouldn't we update the labels?

**Review by abhimanyubabbar — DISMISSED** [review]

File: src/util/prometheus.js (hunk 1 above)
Comment (Jayachand) [reply src/util/prometheus.js:809]: these were moved to end of the list

File: src/util/customTransformer.js [hunk 2]
```diff
@@ -36,9 +38,9 @@ async function runUserTransform(
     new ivm.Reference(async (resolve, ...args) => {
       try {
         const fetchStartTime = new Date();
-        const res = await fetchWithDnsWrapper(versionId, ...args);
+        const res = await fetchWithDnsWrapper(transformationId, ...args);
```
[the remark this answers is not in the document]
Comment (Jayachand) [reply src/util/customTransformer.js]: added workspaceId too

**Review by abhimanyubabbar — APPROVED** [review]

**Review by kanishkkatara — APPROVED** [review]

**Pull request merged**